apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -752,9 +748,7 @@ export async function updateLti13Scores(
     throw new HttpStatusError(403, 'Invalid assessment.id');
   }
 
-  job.info(`Sending grade data for ${assessment.tid} ${assessment.title}`);
-
-  const token = await getAccessToken(instance.lti13_instance.id);
+  job.info(`Working on assessment ${assessment.title} (${assessment.tid}):`);
```

Comment:
Nit, IMO no need to end this line with a colon:

```suggestion
  job.info(`Working on assessment ${assessment.title} (${assessment.tid})`);
```